app/src/components/common/list_item/index.tsx

```diff
@@ -84,7 +84,7 @@ export const ListItem: React.FC<Props> = (props: Props) => {
       const erc20Service = new ERC20Service(provider, account, collateralToken)
       const { decimals, symbol } = await erc20Service.getProfileSummary()
 
-      const amount = ethers.utils.formatUnits(collateralVolume, decimals)
+      const amount = formatBigNumber(collateralVolume, decimals)
```

Comment:
This doesn't have anything to do with closed markets, but I was viewing many decimals in the home screen.